client/src/components/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,60 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    marginTop: '10rem',
+    marginBottom: '10rem',
+    alignItems: 'center',
+    justifyContent: 'center',
+    minWidth: '20rem',
+  },
+  profileContainer: {
+    alignItems: 'center',
+    minHeight: '60vh',
+    margin: 'auto',
+  },
+  contentWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  title: {
+    marginTop: '5rem',
+  },
+  titleText: {
+    fontFamily: 'Roboto',
+    fontWeight: 'bolder',
+    fontSize: '1.5rem',
```

Comment:
Let's try to avoid changing font style outside of our Material UI `theme` as much as possible. Especially the font family should be set within the theme for ensuring a consistent feel/experience throughout the whole application.

We can usually customize the built-in [typography variants](https://v4.mui.com/customization/typography/#variants) theme and use the `variant` prop in the `Typography` component.

Reply:
Thank you for catching this. I will be using `theme` and `variant` for the rest of the project.